CONTRIBUTING.md

```diff
@@ -0,0 +1,150 @@
+# Contributing
+
+## Clone the repo
+
+```bash
+git clone git@github.com:DataDog/build-plugin.git
+```
+
+## Install dependencies
+
+<details>
+<summary>Use Volta (recommended)</summary>
+
+```bash
+brew install volta
+volta setup
+volta install node yarn
+```
+
+</details>
+
+<details>
+<summary>Or install Yarn yourself</summary>
+
+```bash
+brew install yarn
+```
+
+</details>
+
+> [!NOTE]
+> Do not worry about the version, it's dealt with by the repo.
+
+Ensure dependencies are up to date in the repository.
+
+```bash
+cd build-plugins
```

Comment:
Related to https://github.com/DataDog/build-plugin/pull/67/files#r1616167814

I think it's fair to assume that you are in the right folder. Maybe we can drop this `cd`